### PR TITLE
add statemanagement data and illo

### PR DESCRIPTION
**Pull request by zacjones93**
## changes made
- updates [React State Management in 2021](https://egghead.io/courses/react-state-management-in-2021-6732) with dependency data
- updates React State Management in 2021 with course illo
- updates home page and react page with course illo

<details>
<summary>Screen shots</summary>
<br>

Build the portfolio you need to be a badass web developer   egghead io

In-Depth Up-to-Date React Tutorials for 2021  egghead io
React State Management in 2021  egghead io

</details>

### CODE REVIEW

File: src/components/pages/home/homepage-data.ts [hunk 1]
```diff
@@ -395,12 +397,31 @@ There are a number of tried and true methods that will allow you to learn more e
         path:
           '/lessons/react-using-recoil-to-manage-orthogonal-state-in-react-apps-with-david-mccabe?pl=react-state-management-2020-6bec',
       },
+      {
```
Comment (zacjones93) [inline src/components/pages/home/homepage-data.ts:400]: missing a couple of interviews on card for homepage

File: src/components/pages/home/index.tsx [hunk 2]
```diff
@@ -133,11 +135,25 @@ const Home: FunctionComponent = () => {
               </ol>
             </Card>
 
-            <CardVerticalWithStack className="sm:py-3 py-2" data={getStarted} />
-
             <CardVerticalWithStack data={devEssentials} />
 
-            <CardVerticalWithStack data={stateManagement} />
+            <Card resource={tailwind} className="text-center">
```
Comment (zacjones93) [inline src/components/pages/home/index.tsx:140]: moved the 'aside' cards around

File: src/components/pages/home/index.tsx [hunk 3]
```diff
@@ -116,9 +116,11 @@ const Home: FunctionComponent = () => {
             <CardHorizontal resource={reactTeams} />
           </div>
           <aside className="lg:col-span-4 lg:space-y-6 space-y-4">
-            <Card resource={tailwind} className="text-center">
+            <CardVerticalWithStack className="sm:py-3 py-2" data={getStarted} />
+
+            <Card resource={stateManagement} className="text-center">
               <ol className="text-left">
-                {tailwind.resources.map((resource: any, index: any) => {
+                {stateManagement.resources.map((resource: any, index: any) => {
```
Comment (zacjones93) [inline src/components/pages/home/index.tsx:123]: not using CardVerticalWithStack so the image is displayed

File: src/components/search/curated/react/index.tsx [hunk 4]
```diff
@@ -107,11 +107,11 @@ You can find courses below curated just for you whether you're looking for a par
             <div className="flex-shrink-0">
               <Image
                 src={
-                  'https://d2eip9sf3oo6c2.cloudfront.net/series/square_covers/000/000/166/full/EGH_ReactAdvPatterns2_Final.png'
+                  'https://d2eip9sf3oo6c2.cloudfront.net/playlists/square_covers/000/408/538/full/state_management_1000.png'
                 }
                 alt="illustration for state management in react"
-                width={150}
-                height={150}
+                width={200}
```
Comment (zacjones93) [inline src/components/search/curated/react/index.tsx:113]: This illo looked a bit better to me when it was a little bigger

File: src/data/courseDependenciesData.js [hunk 5]
```diff
@@ -3,6 +3,58 @@ import {find} from 'lodash'
 const courseDependencyData = (courseSlug) =>
   find(
     [
+      {
+        id: '408538',
+        type: 'playlist',
+        guid: '6732',
+        slug: 'react-state-management-in-2021-6732',
+        illustrator: {
+          name: 'Maxime Bourgeois',
```
Comment (zacjones93) [inline src/data/courseDependenciesData.js:12]: Maxime is the illustrator!

File: src/data/courseDependenciesData.js [hunk 6]
```diff
@@ -3,6 +3,58 @@ import {find} from 'lodash'
...
+        },
+        freshness: {
+          status: `fresh`,
+          title: `This is a Fresh Resource`,
+          text: `This series digs into the deeper patterns of development with leading experts in the field. It's worth your time.`,
```
Comment (zacjones93) [inline src/data/courseDependenciesData.js:17]: review

File: src/data/courseDependenciesData.js [hunk 7]
```diff
@@ -3,6 +3,58 @@ import {find} from 'lodash'
...
+          asOf: `2021-02-19`,
+        },
+        essentialQuestions: [
+          'How do you know when you need to use a state management library?',
+          'Should all state changes be the result of an event and an action?',
```
Comment (zacjones93) [inline src/data/courseDependenciesData.js:22]: pulled from summaries

File: src/data/courseDependenciesData.js [hunk 8]
```diff
@@ -3,6 +3,58 @@ import {find} from 'lodash'
...
+        ],
+        pairWithResources: [
+          {
+            id: 432472,
+            title: 'Introduction to State Machines Using XState',
```
Comment (zacjones93) [inline src/data/courseDependenciesData.js:27]: thought suggesting state management courses here made sense

**Review by laurosilvacom — APPROVED** [review]

**Pull request merged**